Firebase/Database/Api/FIRDatabase.h

```diff
@@ -29,6 +29,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 FIR_SWIFT_NAME(Database)
 @interface FIRDatabase : NSObject
+  
+/**
+ * The NSObject initializer that has been marked as unavailable. Use the `database` 
+ * method instead
+ *
+ * @return An instancetype instance
```

Comment:
Probably don't need this return annotation in the header comments. Other than that LGTM, but would like @mikelehen to approve